rllib/agents/trainer.py

```diff
@@ -538,7 +543,7 @@ def _setup(self, config):
             self.env_creator = (
                 lambda env_config: NormalizeActionWrapper(inner(env_config)))
 
-        Trainer._validate_config(self.config)
+        self._validate_config(self.config)
```

Comment:
Prefer the static call which is more explicit.

Reply:
done